include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -609,20 +660,15 @@ __parallel_radix_sort(_ExecutionPolicy&& __exec, _Range&& __in_rng)
     using _T = oneapi::dpl::__internal::__value_t<_Range>;
 
     const ::std::size_t __wg_size = oneapi::dpl::__internal::__max_work_group_size(__exec);
-    const ::std::size_t __cunits = oneapi::dpl::__internal::__max_compute_units(__exec);
-
-    ::std::size_t __seg_k = 12;
-    if (__n < __cunits * __wg_size * __seg_k)
```

Comment:
Discussed offline.

Reply:
I removed it to reduce segment size, because I made scan phase more scalable that allowed my to rebalance the work size per phase: scan phase takes more load, but count and reorder (bottleneck) phases take less load.